app/javascript/dashboard/routes/dashboard/settings/integrations/Index.vue

```diff
@@ -17,6 +17,16 @@
               :integration-action="item.action"
             />
           </div>
+          <div class="small-12 columns integration">
+            <integration
+              integration-id="dashboard-apps"
+              integration-logo="dashboard-apps.svg"
+              integration-name="Dashboard apps"
+              integration-description="Dashboard apps lets you add custom applications right into chatwoot dashboard, using the conversation context that chatwoot provides. You can read more about it in the documentation."
```

Comment:
Use i18n.

Reply:
Dashboard Apps allow organizations to embed an application inside the Chatwoot dashboard to provide the context for customer support agents. This feature allows you to create an application independently and embed that inside to provide user information, their orders, or their previous payment history.